pkg/configtx/signer.go

```diff
@@ -130,60 +130,50 @@ func (s *SigningIdentity) SignConfigUpdateEnvelope(configUpdate *cb.ConfigUpdate
 		Signatures:   signatures,
 	}
 
-	signedEnvelope, err := s.createSignedEnvelope(cb.HeaderType_CONFIG_UPDATE, configUpdate.ChannelId, configUpdateEnvelope)
+	signedEnvelope, err := s.newSignedEnvelope(cb.HeaderType_CONFIG_UPDATE, configUpdate.ChannelId, configUpdateEnvelope)
 	if err != nil {
 		return nil, fmt.Errorf("failed to create signed config update envelope: %v", err)
 	}
 
 	return signedEnvelope, nil
 }
 
-// createSignedEnvelope creates a signed envelope of the desired type and signs it.
-func (s *SigningIdentity) createSignedEnvelope(txType cb.HeaderType, channelID string, envelope proto.Message) (*cb.Envelope, error) {
-	channelHeader := channelHeader(txType, msgVersion, channelID, epoch)
-
-	signatureHeader, err := s.signatureHeader()
+// newSignedEnvelope creates a signed envelope of the desired type and signs it.
+func (s *SigningIdentity) newSignedEnvelope(txType cb.HeaderType, channelID string, dataMsg proto.Message) (*cb.Envelope, error) {
```

Comment:
could maybe even call this just `signedEnvelope` and just drop the `new` since that seems to be the pattern we're moving towards

Reply:
I'm indifferent either way. I'll let you decide which you prefer, let me know if you want to merge as is or if you want to change it

Reply:
I say we keep it as-is for now.  We can do another pass to make sure we're consistent in our naming of functions after the current set of tasks are merged in.